### PR TITLE
2361 transfer uses shared nuget

### PR DESCRIPTION
**Pull request by tnickelsen**

### CODE REVIEW

File: domains/transfer/Transfer.API/API.IntegrationTests/Transfer/Api/Services/TransferAgreementProposalCleanupServiceTests.cs [hunk 1]
```diff
@@ -94,7 +94,7 @@ public async Task ShouldCreateActivityLogEntry_WhenDeleting()
         dbContext.TransferAgreementProposals.Add(oldInvitation);
         await dbContext.SaveChangesAsync();
 
-        var invitations = await dbContext.RepeatedlyQueryUntilCountIsMet<TransferAgreementProposal>(0);
+        var invitations = await dbContext.RepeatedlyQueryUntilCountIsMet<TransferAgreementProposal>(0, TimeSpan.FromSeconds(30));
```
Comment (ckr123) [inline domains/transfer/Transfer.API/API.IntegrationTests/Transfer/Api/Services/TransferAgreementProposalCleanupServiceTests.cs:97]: Why this change?

**Review by ckr123 — APPROVED** [review]

File: domains/transfer/Transfer.API/API.IntegrationTests/Transfer/Api/Services/TransferAgreementProposalCleanupServiceTests.cs (hunk 1 above)
Comment (tnickelsen) [reply domains/transfer/Transfer.API/API.IntegrationTests/Transfer/Api/Services/TransferAgreementProposalCleanupServiceTests.cs:97]: Because the test is flaky :( Needed to give it more time to complete. Not sure why it's flaky. Maybe it's something to do with us spinning so many docker images up
